moped-etl/arcgis/components_to_agol.py

```diff
@@ -222,6 +254,21 @@ def main(args):
 
             features = all_features[feature_type]
 
+            # Check project status updates for valid or invalid HTML; escape HTML if needed
```

Comment:
@mddilley it looks like this new html check is only applied when using `args.date` and not `args.full`. i'm still reviewing but wanted to give you that heads-up.

Reply:
🤯 thanks for catching that John! i'm fixing right now